lib/workload.hpp

```diff
@@ -127,98 +129,146 @@ ostream& operator<<(ostream& os, const rhs_t& rhs);
 bool operator==(const rhs_t& rhs1, const rhs_t& rhs2);
 bool operator<(const rhs_t& rhs1, const rhs_t& rhs2);
 
+//************************************* WlSpec *************************************//
+
+class WlSpec {
+public:
+    virtual ~WlSpec() = default;
+    virtual bool applies_to_queue(unsigned int queue) const = 0;
+    virtual bool equals(const WlSpec& other) const = 0;
+    virtual bool less_than(const WlSpec& other) const = 0;
+    virtual int type_id() const = 0;
+    virtual std::string to_string() const = 0;
+    virtual bool is_available_for_search() const { return false; }
+};
+
+//typedef variant<Comp, Same, Incr, Decr, Unique> wl_spec_t;
```

Comment:
Delete this.

Reply:
[Remove wl_spec_t typedef](https://github.com/all-things-networking/fperf/pull/27/commits/d2847cc429c311b70854d0788bfd7412ed3e3391)